src/app/modules/submission/email-submission/email-submission.util.ts

```diff
@@ -651,7 +651,7 @@ const getDataCollationFormattedResponse = (
   if (fieldType !== BasicField.Section) {
     return {
       question: getJsonPrefixedQuestion(response),
-      answer,
+      answer: isNaN(Number(answer)) ? `'${answer}'` : answer,
```

Comment:
may also want to do this processing when decrypting in the client side too.
see `decryption.worker` and the `processDecryptedContent` utility

Reply:
Processing on the client side is done in `CsvMergedHeadersGenerator`